docs/sdk/cli.md

```diff
@@ -0,0 +1,34 @@
+# cosmos-sdk-cli 
```

Comment:
Let's name this file `cosmos-sdk-cli.md`? `cli.md` seems to general there are plenty of CLI's in the SDK!

Reply:
Will do this!